internal/factory/sandbox/infra.go

```diff
@@ -48,6 +49,11 @@ func (s *sandbox) InitInfraContainer(serverConfig *libconfig.Config, podContaine
 		return errors.Wrapf(err, "create resolv conf")
 	}
 
+	// Add capabilities from crio.conf if default_capabilities is defined
+	if err := s.infra.SpecSetupCapabilities(&types.Capability{}, serverConfig.DefaultCapabilities); err != nil {
```

Comment:
Do we have an integration test checking this?

Reply:
not for pod level. to be honest, it's not totally clear to me why we set capabilities for the infra container